### PR TITLE
Add Seattle file with stubs

### PR DESCRIPTION
**Pull request by allthedoll**
I'm stubbing out some Seattle recz, yo. 🎉

### CODE REVIEW

**Comment by githubteacher** [comment]
I think you should be more professional, yo. ⚰️

File: allthedoll-seattle.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Seattle Recommendationz
+
+- Item 1
```
Comment (githubteacher) [inline allthedoll-seattle.md]: @allthedoll Item 1 is a terrible recommendation. Can you please be more descriptive? 👎

File: allthedoll-seattle.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- Item 2
```
Comment (githubteacher) [inline allthedoll-seattle.md]: Same story, different line.

File: allthedoll-seattle.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+## Seattle Recommendationz
```
Comment (githubteacher) [inline allthedoll-seattle.md:1]: Recommendations**

**Review by githubteacher — DISMISSED** [review]
@allthedoll This looks great! But I just have a few changes/recommendations. 😄

File: allthedoll-seattle.md (hunk 3 above)
Comment (allthedoll) [reply allthedoll-seattle.md:1]: Deal with it. 🕶

File: allthedoll-seattle.md (hunk 1 above)
Comment (allthedoll) [reply allthedoll-seattle.md]: I updated my recommendations, @githubteacher. I hope they please you. 🍑 🌹

**Review by allthedoll — COMMENTED** [review]
@githubteacher I've added another recommendation... And some emojis. 🌵 🍔 🥇

**Pull request merged**